### PR TITLE
Using Bootstrap - Milestone 2

### PR DESCRIPTION
**Pull request by verissimohenry**
Using Bootstrap - Milestone 2 

working on the following

Column 1

Featured Stories, Sponsored insights / Rankings

Column 2

Top story, My turn, Culture & travel, More stories, Latest news

Column 3

The debate, Opinion, Sponsored insights / Rankings, Free signup

### CODE REVIEW

File: style.css
Comment (teekaytech) [inline style.css]: Keep your CSS rules DRY (Do not Repeat Yourself).
The rules above can be merged to become one. e.g.
```CSS
.bottom-nav a :hover,
.header-bottom .mid-nav :hover {
  color: #f72210;
}
```

Kindly fix this and check other for occurrences where you might be having code repetition.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made laudable progress in this milestone :+1:__

**_Top highlights_**

- You've added all expected contents for this milestone :ok_hand: 
- Your PR title and summary look great :handshake: 
- Your responsiveness is sweet :1st_place_medal: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Kindly push milestone-2 changes to production: ensure that all features added in milestone-2 are being reflected in your live demo.
- [x] You've overused the `!important` Rule. This is a bad CSS practice. Kindly reduce the number of `!important` rules in your CSS file. [Click here](https://speckyboy.com/good-bad-css-practices/) for a review of other good and bad CSS practices.
- [x] The main goal of this project is to learn how to use the Bootstrap framework, specifically its responsive design features: you should use more bootstrap classes in your project.
- [x] On mobile screen sizes, the title `TOP STORY` is hidden under the navbar. Kindly fix this.
- [x] Again, on mobile screen sizes, your page is having a horizontal scroll bar. Kindly check your containers and find a way to eliminate the scroll bar.


* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, wave:

Thank You for submitting this Pull Request. I am thrilled to announce that your project has met all the minimum requirements and now it is approved. :clap:

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**
